path_checkout_sets.go

```diff
@@ -0,0 +1,411 @@
+package openldap
```

Comment:
This is copied from [vault-plugin-secrets-ad](https://github.com/hashicorp/vault-plugin-secrets-ad).